Classes/Fusion/Helper/NodeInfoHelper.php

```diff
@@ -217,17 +217,23 @@ protected function getUriInformation(NodeInterface $node, ControllerContext $con
      */
     protected function getBasicNodeInformation(NodeInterface $node): array
     {
+        $parent = $node->getParent();
+        $isAutoCreated = $node->isAutoCreated();
+        $parentConfiguration = $parent ? $parent->getNodeType()->getFullConfiguration() : [];
+        $isHidableAutoCreatedChild = $parentConfiguration['childNodes'][$node->getName()]['hideable'] ?? false;
```

Comment:
```suggestion
        $isHidableAutoCreatedChild = $parentConfiguration['childNodes'][(string)$node->getNodeName()]['hideable'] ?? false;
```

Reply:
I added that after applying the first suggestion